src/components/20-molecules/datepicker/index.js

```diff
@@ -123,6 +123,13 @@ class AXADatepicker extends NoShadowDOM {
     return 'axa-datepicker';
   }
 
+  static get version() {
+    return [
+      '7.0.2',
+      { 'axa-dropdown': '7.0.2'.replace(/\./g, '-'), 'axa-button': '2.0.15' },
+    ];
+  }
+
```

Comment:
Maybe 2 getter to have a array with same objects. Something like "get version" and "get versionDependencies".